### PR TITLE
instructions for building VirtualBox appliance

### PR DESCRIPTION
**Pull request by reidpr**

### CODE REVIEW

File: doc-src/virtualbox.rst [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+<https://www.virtualbox.org/>`_ virtual machine appliance with Charliecloud and
+Docker pre-installed. This lets you:
+
+  * use Charliecloud on Macs and Windows
+  * quickly try out Charliecloud without following the install procedure
+
+The virtual machine uses CentOS 7 with an ElRepo LTS kernel. We use the
+:code:`kernel.org` mirror for CentOS, but any should work. Various settings
+are specified, but in most cases we have not done an particular tuning, so use
+your judgement, and feedback is welcome. We assume Bash shell.
```
Comment (trandles-lanl) [inline doc-src/virtualbox.rst]: "but in most cases we have not done any particular tuning"

File: doc-src/virtualbox.rst [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+
+Initialize VM
+-------------
+
+Configure *Preferences* -> *Proxy* if needed.
+
+Create a new VM in VirtualBox:
+
+* Memory: 4 GiB
+* Disk: 48 GiB, VDI dynamically allocated
```
Comment (trandles-lanl) [inline doc-src/virtualbox.rst]: These values are highly subjective. For instance, for the TensorFlow demo I used 1GB RAM and 8GB disk and it was sufficient. Perhaps say something like "At least X GB for [Memory|Disk]".

File: doc-src/virtualbox.rst [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+
+Additional non-default settings:
+
+* *System*
+
+  * *Processor*
+
+    * *Processor(s)*: 4
```
Comment (trandles-lanl) [inline doc-src/virtualbox.rst]: See comment near line 41.

File: doc-src/virtualbox.rst (hunk 2 above)
Comment (j-ogas) [reply doc-src/virtualbox.rst]: The first CentOS 7 (minimal) image I made had an 8 GiB disk, I ran out of disk space while running the Charliecloud test suite. 

The image I used all summer was 20 GiB (I believe). Anyway, hope this helps.

**Pull request merged**